### PR TITLE
Feb1_edits

### PR DESCRIPTION
**Pull request by monaghaa**
updating relion info in Alpine software

### CODE REVIEW

File: docs/cloud/cumulus/cumulus.md [hunk 1]
```diff
@@ -9,6 +9,22 @@ The CUmulus service includes access to a Virtual Private Cloud (VPC) which provi
 
 which can be used to host virtual machines and volumes to host workloads.
 
+---
+**&#9432;** **Announcing the retirement of CUmulus** 
+
+In January 2025 CUmulus will reach the end of its warranty and will be retired.  Numerous alternatives to CUmulus are available, including:  
+
+* the National Science Foundation’s [Jetstream2](https://jetstream-cloud.org) service, which is freely available to any US-based researcher through an ACCESS allocation. 
+* commercial cloud options such as Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform which CURC’s [Cloud Foundations Service](https://www.colorado.edu/rc/cloudfoundations) can facilitate access to;
```
Comment (b-reyes) [inline docs/cloud/cumulus/cumulus.md:18]:
```suggestion
* commercial cloud options such as Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform, which CURC’s [Cloud Foundations Service](https://www.colorado.edu/rc/cloudfoundations) can facilitate access to;  
```

File: docs/cloud/cumulus/cumulus.md [hunk 2]
```diff
@@ -9,6 +9,22 @@ The CUmulus service includes access to a Virtual Private Cloud (VPC) which provi
...
+* transferring instances to CS Cloud OpenStack, which is freely available to any CU [faculty/staff](https://www.colorado.edu/cs/facultystaff-resources/computing-resources-faculty-staff) or [student](https://www.colorado.edu/cs/students/computing-resources-students) (with a faculty sponsor); 
+* other hosted services managed by the CU Office of Information technology, including the [OIT Container Platform](https://oit.colorado.edu/services/consulting-professional-services/container-platform), and [virtual or physical servers](https://oit.colorado.edu/services/consulting-professional-services/systems-engineering/pricing) through the [OIT Systems Engineering Team](https://oit.colorado.edu/services/consulting-professional-services/systems-engineering). 
+
+If you presently have a project that uses CUmulus and you wish to continue using cloud computing beyond the end of 2024, please contact rc-help@colorado.edu to discuss alternatives and receive assistance porting your workflow from CUmulus to another service.  Likewise, if you wish to discontinue your CUmulus project, please send a message to rc-help@colorado.edu.
```
Comment (b-reyes) [inline docs/cloud/cumulus/cumulus.md:22]:
```suggestion
If you presently have a project that uses CUmulus and you wish to continue using cloud computing beyond the end of 2024, please contact <rc-help@colorado.edu> to discuss alternatives and receive assistance porting your workflow from CUmulus to another service.  Likewise, if you wish to discontinue your CUmulus project, please send a message to <rc-help@colorado.edu>. 
```

**Review by b-reyes — APPROVED** [review]
Everything looks good. I just added a comma and make rc-help@colorado.edu a link. Please make sure that the `<rc-help@colorado.edu>` makes it a highlighted email. You can directly commit my suggested changes my clicking on commit suggestion.

**Comment by monaghaa** [comment]
Adding Cumulus retirement announcement

**Pull request merged**